### ISSUE TITLE
Please add TOC at the very beginning

### ISSUE THREAD

**Issue by buckle2000** [ncase/remember#17]
I mean this thing:
![default](https://user-images.githubusercontent.com/10850402/48976146-3a85ed80-f047-11e8-90db-121b09398789.png)

TOC should be available at the start of the page.

TOC should be selected by the first <kbd>Tab</kbd> for accessibility's sake.
